paths/comment_reactions/destroy.yaml

```diff
@@ -10,7 +10,7 @@ parameters:
 - "$ref": "../../parameters.yaml#/key_id"
 - "$ref": "../../parameters.yaml#/comment_id"
 - "$ref": "../../parameters.yaml#/id"
-- description: specify the branch to use
+- description: Branch name
```

Comment:
perhaps this can be extracted to `parameters.yaml` and reused elsewhere?

Reply:
Yep, lots of work but probably a good idea